scripts/setup-caas-agents.sh

```
@@ -0,0 +1,108 @@
+  --cdrom ${AGENT_VM_STORAGE_DIR}/discovery.iso \
+  --network network=${LIBVIRT_NETWORK} \
+  --os-variant rhel9.0 \
+  --boot hd,cdrom \
+  --noautoconsole
+
+echo "Agent VM created and booting"
+SSHEOF
+
+# Wait for agent to register
+echo "Waiting for agent to register..."
+retry_until 300 15 '[[ $(oc get agent -n ${AGENT_NAMESPACE} --no-headers 2>/dev/null | wc -l) -gt 0 ]]' || {
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)